coleslaw.asd

```diff
@@ -14,7 +14,8 @@
                :cl-fad
                :cl-ppcre
                :closer-mop
-               :cl-unicode)
+               :cl-unicode
+               :djula)
```

Comment:
If at all possible, I would prefer we load djula in an `eval-when` in the plugin as we do with other plugins that have additional library dependencies. If it is necessary to also initialize/compile templates at that time, the data necessary to do so can be passed into the `enable` method in the plugin and they can be compiled then.